jekyll/_cci2/contexts.md

```diff
@@ -254,6 +264,21 @@ endpoint with the appropriate request body. For this request, replace the
 environment variable name. The request body should include a `value` key 
 containing the plaintext secret as a string.
 
+##### Environment Variable validation
```

Comment:
3 `#`s right? So it's not included as part of `Secure Environment Variable Creation, Deletion, and Rotation`

Reply:
this is 5 `#`s because it falls under `#### Creating Environment Variables`